### DISCUSSION TITLE
Any instructions to install on windows?

### DISCUSSION THREAD

**Discussion by jianlinshi** [qdrant/qdrant#2772] [Q&A] [answered]
I know windows are not popular in dev environments, but that's all we got and all we can get. And docker is not approved to be installed on our servers. Thanks.

**Comment by generall** [comment, marked as answer]
Hi @jianlinshi, along with docker we also provide binary build for different platforms, including windows: https://github.com/qdrant/qdrant/releases/tag/v1.5.1

However the recommended way of running qdrant is still a docker container under linux-based OS
